### PR TITLE
Portfolio deployment accessibility test : module1

### PR DESCRIPTION
**Pull request by yusufyusie**
### Deployment accessibility test milestone: module 1
**In this milestone of your portfolio website evaluation: we get and**
 "Page titles - no issues found"

- "Image text alternatives- no issues found"
- "text headings -  h1 is not set as similar as title and h1 headings are not settled in the similar format [Link ](https://github.com/yusufyusie/yusufyusie.github.io/blob/89558f5d3d6a48a4595d5fdab2544c0741bec6ba/index.html#LL7C4-L7C40)  and there is same h1 text and styles more than header that  is h1 duplication"[Link ](https://github.com/yusufyusie/yusufyusie.github.io/blob/89558f5d3d6a48a4595d5fdab2544c0741bec6ba/index.html#LL60C11-L62C16)
- "Color contrast -   no issues found"
- "Resize -  no issues found"
- "Moving content -  no issues found"
- "Multimedia -  no issues found this because there is no video on the live demo of the portfolio"
- "Basic structure of the page check  - there one thing that to is changed because when the website display resolution changes some of the text overlaps on the background image that results in some uncomfortable issues so there must be some editing on h2 style" [Link ](https://github.com/yusufyusie/yusufyusie.github.io/blob/89558f5d3d6a48a4595d5fdab2544c0741bec6ba/styles.css#LL33C3-L37C20)

### CODE REVIEW

File: README.md
Comment (Ofarouq310) [inline README.md:1]: - [ ] Kindly take a look at your `README`, it looks like it is two `README` files overlapping and mixed up with each other.

File: index.html [hunk 1]
```diff
@@ -35,8 +35,8 @@ <h3>
                     </a>
                 </li>
                 <li>
-                    <a href="#footer-container-id" class="link-contact">
-                        Contact
+                    <a href="#footer-container-id" class="link-contact"><p>
+                        Contact <p/>
                     </a>
```
Comment (Ofarouq310) [inline index.html]: [OPTIONAL] 
- [ ] Kindly close your paragraph tag correctly.
```suggestion
                    <a href="#footer-container-id" class="link-contact"><p>
                        Contact </p>
                    </a>
```

**Review by Ofarouq310 — CHANGES_REQUESTED** [review]
Hi @yusufyusie, @paulshaun-kdtk & @lewisjn-arch, ✋  

Good job so far! 💪 
There are some issues that you still need to work on to go to the next project but you are almost there! 🥈 

giphy

## Highlights
✅ Descriptive PR title.

## Required Changes ♻️

- [ ] Kindly make sure that any changes or accessibility issues you have fixed, you should link to the corresponding line of code in the Pull Request changes, to get a better understanding of how to do so, please refer to this [tutorial](http://www.chasewoodford.com/blog/linking-highlighted-lines-code-github/#:~:text=To%20link%20to%20multiple%20lines,php%23L117%2DL148).

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ofarouq310 — CHANGES_REQUESTED** [review]
Hi @yusufyusie, @paulshaun-kdtk & @lewisjn-arch, ✋  

Good job so far! 💪 
There are some issues that you still need to work on to go to the next project but you are almost there! 🥈 

giphy

## Highlights
✅ Descriptive PR title.

## Required Changes ♻️

- [ ] You still did not add  the accessibility changes links in your pull request, you should have added the links in your pull request beside their corresponding point, for example, let's say you made changes concerning the second point, which is "text headings", so besides that point in your PR, you should add your link, e.g.:
    - "text headings - h1 is not set as similar as title and h1 headings are not settled in the similar format and styles" - changes made on line [`64`](https://github.com/yusufyusie/yusufyusie.github.io/blob/b64c7fdd5d3a30fa73206e937846983a2a526c0a/index.html#LL64C10-L64C10).
    - That is how it is written:         
      image


- [ ] Kindly make sure to check your `README` file because it is more than one `README combined together.
  
  image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lewisjn-arch** [comment]
#### This third review request has:


- [ ] Text headings - h1 tag was set on text that isn't the main semantic tag of the page, changed tag to h2" - changes made on line `60` to `62`

https://github.com/yusufyusie/yusufyusie.github.io/blob/developer/index.html#L60-L62


- [ ] Updated README.md file and added collaborators.

**Review by abdallahmalima — APPROVED** [review]
Hi @yusufyusie , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Report includes a check about page titles.✔️
- Report includes a check about image text alternatives.✔️
- Report includes a check about colour contrast.✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._✔️

**Pull request merged**